web/src/composebox_typeahead.ts

```diff
@@ -1129,7 +1128,7 @@ export function content_typeahead_selected(
                     // use markdown link syntax
                     beginning += topic_link_util.get_fallback_markdown_link(item.name);
                 } else {
-                    beginning += "#**" + item.name + "** ";
+                    beginning += "#**" + item.name + ">";
```

Comment:
I think you can remove the whole `if (event && event.key === ">") {` block above, since we'll now always add the ">".